src/app.scss

```diff
@@ -1,3 +1,5 @@
+@import 'carbon-components/scss/globals/scss/typography.scss';
+@import 'carbon-components/scss/globals/scss/vars.scss';
 @import './components/TutorialHeader/tutorial-header.scss';
```

Comment:
You're missing `@import 'carbon-components/scss/globals/scss/vendor/@carbon/layout/scss/breakpoint.scss';`